tests/formats_test/selinux_test.py

```diff
@@ -13,14 +13,13 @@ class SELinuxContextTestCase(unittest.TestCase):
     """
 
     def setUp(self):
-        if not blivet.flags.selinux:
-            self.skipTest("SELinux disabled.")
         self.installer_mode = blivet.flags.installer_mode
         super(SELinuxContextTestCase, self).setUp()
 
     @patch("blivet.util.mount", return_value=0)
     @patch.object(fs.FS, "_pre_setup", return_value=True)
     @patch("os.access", return_value=True)
+    @patch("selinux.lsetfilecon")
```

Comment:
You're already patching `selinux.lsetfilecon` below (from the original commit, I think). Adding this patch call for the whole method shouldn't be necessary unless there is something going on that needs to be better understood.